include/regs.h

```diff
@@ -58,15 +58,26 @@
 #define R_FB_FILTER_ENV_COLOR(c)                 SREG(85 + (c))
 #define R_ENABLE_FB_FILTER                       SREG(88)
 #define R_PAUSE_MENU_MODE                        SREG(94)
+#define R_CAM_XZ_OFFSET_UPDATE_RATE              OREG(2)
+#define R_CAM_Y_OFFSET_UPDATE_RATE               OREG(3)
+#define R_CAM_FOV_UPDATE_RATE                    OREG(4)
 #define R_CAM_MAX_PITCH                          OREG(5)
-#define R_CAM_DEFAULT_PITCH_UPDATE_RATE_INV      OREG(7)
+#define R_CAM_R_UPDATE_RATE_INV                  OREG(6)
+#define R_CAM_PITCH_UPDATE_RATE_INV              OREG(7)
+#define R_CAM_SLOPE_Y_ADJ_AMOUNT                 OREG(9)
 #define R_CAM_PITCH_FLOOR_CHECK_NEAR_DIST_FAC    OREG(17)
 #define R_CAM_PITCH_FLOOR_CHECK_FAR_DIST_FAC     OREG(18)
 #define R_CAM_PITCH_FLOOR_CHECK_OFFSET_Y_FAC     OREG(19)
 #define R_CAM_PITCH_FLOOR_CHECK_NEAR_WEIGHT      OREG(20)
 #define R_CAM_DEFAULT_ANIM_TIME                  OREG(23)
+#define R_CAM_UPDATE_RATE_STEP_SCALE_XZ          OREG(25) // name not ideal
+#define R_CAM_UPDATE_RATE_STEP_SCALE_Y           OREG(26) // name not ideal
```

Comment:
imo no need to comment on this, if someone has a better idea they will suggest it/pr it. its just not something we really do. but its not the biggest deal

Reply:
Agree with this. If anything a comment explaining why it isn't ideal would be more beneficial.